SlackBotPrototype/API.Tests/EmbeddedResourceTestBase.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace API.Tests
+{
+	public class EmbeddedResourceTestBase
+	{
+
+		protected string GetFromResource(string fileName)
+		{
+			if (fileName == null)
+				throw new ArgumentNullException(nameof(fileName));
+
+			var assembly = Assembly.GetExecutingAssembly();
+			var resourceName = assembly.GetManifestResourceNames().First(x => x.Contains(fileName));
+
+			using (var file = assembly.GetManifestResourceStream(resourceName))
+			{
+				if (file == null)
+					Assert.Fail();

```

Comment:
i would `throw` here, instead of using `Assert` not sure how it works in c# but some test suites will report test failures different from unexpected exceptions, and not having an expected fixture file is probably in the latter category